02_bubble_sort:selva/bubblesort.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(items []int) []int {
```

Comment:
This a copying flavour of quicksort, not bubble sort. I've made recommendations below, anyway. I'd suggest you just rename it, since it's better than bubble sort and gets you the extra bonus points.

Double extra bonus points if you rework it to avoid copying, except at the top level, as required.